### PR TITLE
Update Entra ID SSO steps

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3286

### CODE REVIEW

File: astro/configure-idp.md [hunk 1]
```diff
@@ -164,56 +164,22 @@ To integrate Azure as your IdP for Astro you must have:
...
+Submit a request to [Astronomer support](https://cloud.astronomer.io/support) to connect Astro to Microsoft Entra ID. In your request, provide the following information:
 
-    - **Connection type**: Select **Microsoft Entra ID**.
-    - **SSO Domain(s)**: Enter the verified domain(s) that you want to map to Microsoft Entra ID.
-    - **Automatic Membership**: Set the default role for users who join your Organization through Microsoft Entra ID and without an explicit invite from Astro.
-    - **Microsoft Entra ID Domain**: Retrieve this value from your Microsoft Entra ID directory's overview page in the Microsoft Azure portal.
-    - **Application (client) ID**: Retrieve this from the **Overview** page of your Microsoft Entra ID application.
-    - **Client ID and Client secret**: Enter the values you coped from [Step 2: Create a client secret](#step-2-create-a-client-secret)
+- Your PEM certificate
+- ["appropriate copied URLs from Microsoft Entra admin center"??]
```
Comment (jwitz) [inline astro/configure-idp.md:178]: @chomipi88 I tried reverse engineering this setup from the Microsoft doc and this line stuck out to me. What other info does support need from the user?

**Comment by chomipi88** [comment]
User can also be at least a [Cloud Application Administrator](https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference#cloud-application-administrator) to perform the setup.

https://learn.microsoft.com/en-us/entra/identity/saas-apps/astro-tutorial#:~:text=managed%20SaaS%20apps.-,Sign%20in%20to%20the%20Microsoft%20Entra%20admin%20center%20as%20at%20least%20a%20Cloud%20Application%20Administrator.,-Browse%20to%20Identity

File: astro/configure-idp.md [hunk 2]
```diff
@@ -164,56 +164,22 @@ To integrate Azure as your IdP for Astro you must have:
...
 
-Follow [Microsoft Documentation](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app) to register a new app. When configuring the application, set the following values:
+Follow the [Microsoft documentation](https://learn.microsoft.com/en-us/entra/identity/saas-apps/astro-tutorial#add-astro-from-the-gallery) to add Astro from the gallery to your list of managed SaaS applications.
 
-- **Name** and **Supported account types**: Set these according to your organization's needs.
-- **Redirect URI**: Select **Web** and specify `https://auth.astronomer.io/login/callback`.
+#### Step 2: Configure SSO on Astro
 
-#### Step 2: Create a client secret
+Follow the steps in [Microsoft documentation](https://learn.microsoft.com/en-us/entra/identity/saas-apps/astro-tutorial#configure-microsoft-entra-sso) to configure Microsoft Entra SSO to connect to Astro. Be sure to download the PEM certificate from teh **SAML Signing Certificate** option for the next step.
```
Comment (chomipi88) [inline astro/configure-idp.md:171]:
```suggestion
Follow the steps in [Microsoft documentation](https://learn.microsoft.com/en-us/entra/identity/saas-apps/astro-tutorial#configure-microsoft-entra-sso) to configure Microsoft Entra SSO to connect to Astro. Be sure to download the PEM certificate from the **SAML Signing Certificate** option for the next step.
```

File: astro/configure-idp.md [hunk 3]
```diff
@@ -164,56 +164,22 @@ To integrate Azure as your IdP for Astro you must have:
 
 #### Step 1: Register Astro as an application on Azure
 
-Follow [Microsoft Documentation](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app) to register a new app. When configuring the application, set the following values:
+Follow the [Microsoft documentation](https://learn.microsoft.com/en-us/entra/identity/saas-apps/astro-tutorial#add-astro-from-the-gallery) to add Astro from the gallery to your list of managed SaaS applications.
 
-- **Name** and **Supported account types**: Set these according to your organization's needs.
-- **Redirect URI**: Select **Web** and specify `https://auth.astronomer.io/login/callback`.
+#### Step 2: Configure SSO on Astro
```
Comment (chomipi88) [inline astro/configure-idp.md:169]: For this step the customer we'll need to grab the below info from Astro UI:

AUDIENCE URI (SP ENTITY ID)
SINGLE SIGN ON URL

Similar to: Copy the Single Sign-On URL and Audience URI (SP ENTITY ID) for Step 8.

File: astro/configure-idp.md [hunk 4]
```diff
@@ -164,56 +164,22 @@ To integrate Azure as your IdP for Astro you must have:
...
 
-#### Step 2: Create a client secret
+Follow the steps in [Microsoft documentation](https://learn.microsoft.com/en-us/entra/identity/saas-apps/astro-tutorial#configure-microsoft-entra-sso) to configure Microsoft Entra SSO to connect to Astro. Be sure to download the PEM certificate from teh **SAML Signing Certificate** option for the next step.
 
-Follow [Microsoft documentation](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-credentials) to create a client secret for your new application. Make note of the client ID and secret value for Step 4.
+#### Step 3: Send configuration information to Astronomer support
```
Comment (chomipi88) [inline astro/configure-idp.md:173]: Step 3 is self service in the Astro UI. Similar to: https://docs.astronomer.io/astro/configure-idp?tab=OneLogin#configure-your-identity-provider:~:text=Return%20to%20the%20Cloud%20UI.%20In%20the%20configuration%20screen%20for%20your%20SAML%20connection%2C%20configure%20the%20following%20values%3A

File: astro/configure-idp.md (hunk 1 above)
Comment (chomipi88) [reply astro/configure-idp.md:178]: This is the "IDENTITY PROVIDER SINGLE SIGN-ON URL" the user will need to input in the Astro UI

File: astro/configure-idp.md [hunk 5]
```diff
@@ -164,56 +164,22 @@ To integrate Azure as your IdP for Astro you must have:
...
-    - **Application (client) ID**: Retrieve this from the **Overview** page of your Microsoft Entra ID application.
-    - **Client ID and Client secret**: Enter the values you coped from [Step 2: Create a client secret](#step-2-create-a-client-secret)
+- Your PEM certificate
+- ["appropriate copied URLs from Microsoft Entra admin center"??]
 
-5. If you already completed [Step 1: Register Astro as an application on Azure](#step-1-register-astro-as-an-application-on-azure), skip the Cloud UI instructions to configure a Redirect URI.
-6. Click **Create**. Your Microsoft Entra ID integration appears as an entry in **SSO Configuration**.
-7. In **SSO Configuration**, click **Activate**. You are redirected to Microsoft Entra ID to test your configuration. After you have successfully authenticated, you are redirected to Astro.
-8. Click **Activate SSO**.
+After you submit your request, Astronomer support creates the SSO connection and notifies you when the connection is ready to use.
```
Comment (chomipi88) [inline astro/configure-idp.md:180]: All self-service. Support not required.

File: astro/configure-idp.md (hunk 4 above)
Comment (jwitz) [reply astro/configure-idp.md:173]: Not sure who we reach out to about this, but the Entra ID Astro docs are out of date and reference contacting support: https://learn.microsoft.com/en-us/entra/identity/saas-apps/astro-tutorial#configure-astro-sso

File: astro/configure-idp.md [hunk 6]
```diff
@@ -158,60 +158,44 @@ This section provides setup steps for setting up Microsoft Entra ID as your IdP
...
+#### Step 4: Finalize the SSO connection in Astro
 
-#### Step 4: Create an SSO connection to Microsoft Entra ID
+1. Return to the Cloud UI. In the configuration screen for your SSO connection, configure the following values:
 
-1. Assign yourself to Astro from Microsoft Entra ID. See [Assign users and groups to an Application](https://learn.microsoft.com/en-us/entra/identity/enterprise-apps/assign-user-or-group-access-portal?pivots=portal).
-2. In the Cloud UI, click your Workspace name in the upper left corner, click **Organization Settings**, then click **Authentication**.
-3. In the **SSO Configuration** menu, click **Configure SSO**.
-4. Configure the following values for your connection:
+    - **PEM Certificate**: Enter the PEM certificate you downloaded in Step 3.
```
Comment (jwitz) [inline astro/configure-idp.md:195]: @chomipi88 Does the Cloud UI update to show **PEM Certificate**? The only SAML configs I can find use X.509.

Comment (chomipi88) [reply astro/configure-idp.md:195]: @jwitz The user will paste the certificate in the X.509 CERTIFICATE section of the SSO configuration in Cloud UI

**Review by chomipi88 — APPROVED** [review]

File: astro/configure-idp.md (hunk 6 above)
Comment (jwitz) [inline astro/configure-idp.md:195]:
```suggestion
    - **X.509 Certificate**: Enter the PEM certificate you downloaded in Step 3.
```

**Pull request merged**